pkg/asset/k8s.go

```diff
@@ -163,7 +163,7 @@ func secretFromAssets(name, namespace string, assetNames []string, assets Assets
 	})
 }
 
-func mustCreateAssetFromTemplate(name string, template []byte, data interface{}) Asset {
+func MustCreateAssetFromTemplate(name string, template []byte, data interface{}) Asset {
```

Comment:
Our templates sit inside an internal pkg. There is no easy way to ONLY load in one recovery etcd template. Alternatively, I can put a special loading logic inside Asset pkg (WriteOutRecoveryEtcd or something). But I feel it is less idea.

Also I actually want to put the recovery etcd template inside the recovery pkg.